components/mpas-ocean/src/Registry.xml

```diff
@@ -2419,6 +2579,13 @@
 			 description="time tendency of sea-surface height"
 			 packages="forwardMode"
 		/>
+                <var name="tendTKE" type="real" dimensions="nVertLevels nCells Time" units="m^2 s^{-3}" name_in_code="tendTKE"
+                         description="time tendency of turbulence kinetic energy"
+                         packages="forwardMode"
+                />
+                <var name="tendglsPsi" type="real" dimensions="nVertLevels nCells Time" units="m^2 s^{-4}" name_in_code="tendglsPsi"
+                         description="time tendency of psi variable from Umlauf and Burchard (2003)"
+                />
```

Comment:
Same as above.  Either the `name_in_code` should be `tke` and `gslPsi` for consistency.

Reply:
Add to `nonhydrostaticPKG`